rfc/rfc014-authorization-credential.md

```diff
@@ -0,0 +1,219 @@
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 014 | Nedap |
+|  | June 2021 |
+
+## Nuts Authorization Credential
+### Abstract
+
+A Nuts Authorization Credential describes which data an actor may access. 
+It is scoped to a specific combination of custodian, actor and service (Bolt). 
+The credential also contains the legal base on which data access may occur. 
+This allows this credential to be used for explicit consent from a patient but also in cases where consent is implied.
+A patient can consent to a referral to a specialist. This implies that relevant data is accessible to that specialist.
+This RFC adds the requirement for Bolts to define an access policy. 
+A resource server, hosted by a custodian, will use the policy and the credential to grant access to a specific resource. 
+The credential is currently only usable for FHIR based services.
+
+### Status
+
+This document is currently a draft.
+
+### Copyright Notice
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+Personal data (this includes medical data) is protected under the GDPR and various local legislation. 
+It's common in healthcare to receive care from multiple care organizations. 
+These organizations may not share data amongst them without a valid legal reason. 
+The GDPR sums up several legal reasons to share data. Most common are local legislation and explicit consent. 
+When data is shared between two organizations, this only matters to those organizations (and the patient). 
+Others are not allowed to see this relation. 
+Simply knowing where someone receives care may give away the illness of the patient.
+The type of data that is shared should also be limited to what is needed by the actor.
+
+There are various reasons why data can be shared. 
+The process of determining a valid reason happens at different places and at different times:
+
+- A patient at the GP's office wants the GP to view data from the hospital.
+- A patient consents to a specific treatment or test and this treatment/test requires specific data.
+- An elderly receiving home care wants his/her GP to be able to review data from the home care organization.
+- A patient is discharged from the hospital and needs to undergo physical therapy. 
+- Etc.
+
+This RFC builds upon [RFC011](rfc011-verifiable-credential.md).
+
+## 2. Terminology
+
+* **DID**: [Decentralized Identifiers](https://www.w3.org/TR/did-core/).
+* **Policy**: A security policy defined by a Bolt. It describes the access to and operations on resources that are allowed. 
+* **VC**: [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+Other terminology comes from the [Nuts Start Architecture](rfc001-nuts-start-architecture.md#nuts-start-architecture).
+
+## 3. CredentialSubject
```

Comment:
Perhaps rename this to `VC structure` or something, since it coveres more than the CredentialSubject propertie of the VC.

Reply:
Renamed to `Credential data model` (term used by w3c specification)